vexctl.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: vexctl
+  version: 0.2.3
+  epoch: 0
+  description: A tool to create, transform and attest VEX metadata
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - go
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
```

Comment:
```suggestion
```
This is automagically added by the [go/install](https://github.com/chainguard-dev/melange/blob/a4590d130eb51801d2819183e5cdc19fbcd079db/pkg/build/pipelines/go/install.yaml#L5) pipeline so can be removed

Reply:
Ohhh, too cool. Will remove.

Reply:
See commit b17865a